Dockerfile

```diff
@@ -2,17 +2,18 @@ FROM registry.access.redhat.com/ubi8/ubi:latest AS build
 
 ARG OPENSHIFT_VERSION
 ENV OPENSHIFT_VERSION=${OPENSHIFT_VERSION}
```

Comment:
So if we're not using this OPENSHIFT_VERSION arg/env any more, they can be removed, right?

Reply:
Good call, will remove.